test/sql/test_computed.py

```diff
@@ -0,0 +1,54 @@
+# coding: utf-8
+from sqlalchemy import Column, Computed, Integer, MetaData, Table
+from sqlalchemy.exc import ArgumentError
+from sqlalchemy.schema import CreateTable
+from sqlalchemy.testing import (
+    AssertsCompiledSQL,
+    combinations,
+    fixtures,
+    assert_raises_message,
+)
+
+
+class DDLComputedTest(fixtures.TestBase, AssertsCompiledSQL):
+    __dialect__ = "default"
+
+    @combinations(
+        ("no_persisted", "", Ellipsis),
```

Comment:
**mike bayer** (zzzeek) wrote:

we're still doing Python 2 so this has to be something that works in py2k also.

Reply:
I'll change this.
The logic of the test is that we have 4 cases:
- Persisted=true
- Persisted=false
- Persisted=None
- Persisted not provided

I use Ellipsis for the last case, to ignore the value passed to the function, and call only `Computed`